prow/repoowners/repoowners.go

```diff
@@ -84,6 +84,14 @@ type cacheEntry struct {
 	owners  *RepoOwners
 }
 
+func (entry cacheEntry) isValid(mdYaml bool) bool {
+	return entry.sha != "" && entry.aliases != nil && entry.owners != nil && entry.owners.enableMDYAML == mdYaml
+}
+
+func (entry cacheEntry) isValidOwners(mdYaml bool) bool {
```

Comment:
Is this more like `matchesMdYaml`? We're checking that the setting in the entry (`entry.owners.enableMDYAML`) is the same as what we have (`mdYaml`)

Reply:
fixed